core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div>
+        <a ng-href="/collection/{{collectionId}}" class="conversation-skin-back-to-collection" translate="I18N_PLAYER_BACK_TO_COLLECTION"></a>
```

Comment:
Does this have something to do with the fact that collectionId is JSON (at least I think it is based on a JSON.stringify call on line 777 of ConversationSkinDirective.js), and that's why it doesn't get rendered?

On a side note, is this the right approach? I used it because I figured it was a way to return to the unique collection the user is currently on.